CONTRIBUTING.md

```diff
@@ -282,74 +279,82 @@ This will allow you to see a local version of the specification.
 The local address will be `http://127.0.0.1:8000`.
 You may enter that into your browser and this will bring up the specification!
 
-## Fixing Travis Remark errors
+## Fixing markdown style errors
 
 We use a linter called [Remarkjs](https://github.com/remarkjs/remark-lint) to
 ensure all of our Markdown documents are consistent and well-styled.
-This commonly produces errors, which are flagged by [Travis CI](https://travis-ci.org/),
+This commonly produces errors, which are flagged by [GitHub Actions](https://github.com/features/actions),
 a continuous integration service.
-When Travis returns an error, use the following process to resolve the issue:
+When GitHub Actions returns an error, use the following process to resolve the issue:
 
-#### 1. Install NodeJS / npm
+### 1. Install NodeJS / npm
 
 We use a markdown linter written in Javascript. To run command Javascript tools
 on the command line, please [download and install](https://nodejs.org/en/download/)
 NodeJS.
 
-#### 2. Install Remark-CLI and our style guide
+### 2. Install Remark-CLI and our style guide
 
 Remark-CLI can be installed via [npm](https://www.npmjs.com/), which is part of
 the NodeJS distribution.
 
 To install the packages we use for our style guide, the following command will
 work on most command lines:
 
-```
+```shell
 npm install `cat npm-requirements.txt`
 ```
 
 The equivalent command on PowerShell is:
 
-```
+```shell
 npm install @(cat npm-requirements.txt)
 ```
 
-#### 3. Fix the flagged document
+### 3. Find documents that are failing the check
 
-Please go to the directory where the flagged file is and run remark like this:
+Run the following from the root directory of `bids-specification`:
 
+```shell
+npx remark ./src/*.md ./src/*/*.md
 ```
-remark flagged_file.md -o flagged_file_fixed.md
+
+### 4. Fix the flagged document
+
+Please go to the directory where the flagged file is and run remark like this:
+
+```shell
+npx remark flagged_file.md -o flagged_file_fixed.md
 ```
 
 Please confirm this has fixed the file. To do this, please run this:
 
-```
-remark flagged_file_fixed.md --frail
+```shell
+npx remark flagged_file_fixed.md --frail
 ```
 
 This command will indicate whether this file now conforms to the style guide.
 If it passes, replace `flagged_file.md` with the contents of
 `flagged_file_fixed.md`, add and commit the change:
 
-```
+```shell
 mv flagged_file_fixed.md flagged_file.md
 git add flagged_file.md
 git commit -m 'STY: Fixed Markdown style'
 ```
 
```

Comment:
```suggestion

NOTE: 

Using `remark` to fix some linting errors might introduce some additional changes:

- changing unordered list from using `-` to using `*`
- changing ordered list from using `1.` to actually using the number of the item
- changes literal hyperlinks URLs from `[URL](URL)` to `<URL>`
- in some instances, it will "escape" all `_` and `&` with a `\` in all the URLs.

You might have to revert those or use [interactive staging](https://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging) to make sure you only commit the right hunks of code.

```

Reply:
Tempted to add this until we find a way to make sure `remark` is not over zealous.